src/support_bank_python/supportbank.py

```diff
@@ -0,0 +1,73 @@
+import csv
+
+class Account:
+    def __init__(self, name):
+        self.name = name
+        self.balance = 0
+        self.transactions = []
+    
+    def update_balance_for_from(self, amount):
+        self.balance -= amount
+
+    def update_balance_for_to(self, amount):
+        self.balance += amount
+
+    def update_transactions(self, transaction):
+        self.transactions.append(transaction)
+
+
+class Transaction:
+    def __init__(self, date, fromAcc, toAcc, narrative, amount):
+        self.date = date
+        self.fromAcc = fromAcc
+        self.toAcc = toAcc
+        self.narrative = narrative
+        self.amount = amount
+
+def poundsToPence(pounds):
+    return round(pounds*100, 0)
+
+def penceToPounds(pence):
+    return round(pence/100, 2)
+
+def readfile(filePath, accountLedger, transactionLedger):
+    with open(filePath, mode='r') as file:
+        csvFile = csv.DictReader(file)
+        for line in csvFile:
+            amount = poundsToPence(float(line['Amount']))

```

Comment:
rather than using float could consider using decimal and then converting to an int once in pence to remove any risk of rounding errors introduced by floating point numbers